mne/viz/_brain/_brain.py

```diff
@@ -177,6 +177,8 @@ def __init__(self, subject_id, hemi, surf, title=None,
 
         if isinstance(background, str):
             background = colorConverter.to_rgb(background)
+        if foreground is None:
+            foreground = "white"
```

Comment:
We should set it to white or black automatically depending on the luminance of the `background`, see [how it's done in PySurfer](https://github.com/nipy/PySurfer/blob/master/surfer/viz.py#L563-L564)